atox/src/main/kotlin/ui/AvatarImageView.kt

```diff
@@ -0,0 +1,100 @@
+package ltd.evilcorp.atox.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import androidx.constraintlayout.widget.ConstraintLayout
+import kotlin.math.pow
+import kotlin.math.sqrt
+import ltd.evilcorp.atox.R
+import ltd.evilcorp.atox.databinding.AvatarImageLayoutBinding
+
+class AvatarImageView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) :
+    FrameLayout(context, attrs, defStyleAttr) {
+
+    companion object {
+        private const val STATUS_INDICATOR_SIZE_RATIO_WITH_AVATAR = 12f / 50
```

Comment:
You don't need a companion object for things like this in Kotlin. You can just put a `private const val THING = 123` at the file scope.

Reply:
Hah. We already merged such a thing in AvatarFactory. But this PR removes it anyway.

Reply:
Ah the difference is that in AvatarFactory it wasn't private

Reply:
No, I just didn't think to comment on it right then. :x I think in general in aTox we have ~0 companion objects unless they're methods that should be able to be called without an instance of the class.